UI.AutomationTests/TestData/HearingTestData.cs

```diff
@@ -242,6 +242,65 @@ public static BookNewHearingRequestV2 CreateNewRequestDtoWithOnlyAJudge(bool rem
         return request;
     }
     
+    public static BookNewHearingRequestV2 CreateNewRequestDtoWithJudgeAndRep(bool remote = false,
```

Comment:
Any chance you could refactor this so the other methods that return a `BookNewHearingRequestV2` use a private shared method?